components/webhook-mgt/org.wso2.carbon.identity.webhook.metadata/src/main/resources/risc-event-profile.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Let's not add risc

Reply:
fixed with https://github.com/wso2/carbon-identity-framework/pull/6782/commits/18d9e9a65adb6dc98ea02b0021b59c5d0804bcaf